index.html

```diff
@@ -0,0 +1,158 @@
+          <li class="nav-link"><a href="#">Join</a></li>
+          <li class="nav-link"><a href="#">Sponsor</a></li>
+          <li class="nav-link"><a href="#">News</a></li>
+          <li class="nav-link"><a href="#">YWCC Campaign</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+
+  <div class="home-page active" id="home-page">
+    <section class="home">
+      <h1 class="p-title">YEARLY COMEDIANS WORLD CONFERENCE</h1>
+      <p class="description">A joyful reunion of the most popular living comedians from all over the globe, to celebrate comedy, taking place this December in Spain.</p>
+      <div class="date-and-place">
+        <p class="date">2022.12.15(THU) - 16(FRI)</p>
+        <p class="place">@ Camp Nou, C. d'Arístides Maillol, 12, 08028 Barcelona, Spain</p>
+      </div>
+    </section>
+  
+    <section class="main-program">
+      <div class="s-title">
+        <h2>Main Program</h2>
+        <hr>
+      </div>
+  
+      <ul class="activities">
+  
+        <li class="activity">
+          <img src='/media/public-speaker-icon.png' alt="" class="icon">
+          <h3 class="title">Lecturee</h3>
+          <p class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero esse eveniet est.</p>
+        </li>
+  
+        <li class="activity">
+          <img src="/media/board-icon.png" alt="" class="icon">
+          <h3 class="title">Conference 
+            Exhibition</h3>
+          <p class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+        </li>
+  
+        <li class="activity">
+          <img src="/media/chat-icon.png" alt="" class="icon">
+          <h3 class="title">Forum</h3>
+          <p class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis unde</p>
+        </li>
+  
+        <li class="activity">
+          <img src="/media/workshop-icon.png" alt="" class="icon">
+          <h3 class="title">Workshop</h3>
+          <p class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reiciendis?</p>
+        </li>
+  
+        <li class="activity">
+          <img src="/media/networking-icon.png" alt="" class="icon">
+          <h3 class="title">Conference Ignite</h3>
+          <p class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+        </li>
+  
+      </ul>
+  
+      
+      <div class="join-btn">Join YW Comedians Conference 2022</div>
+
+      <a href="#" class="see-program">SEE THE WHOLE PROGRAM</a>
+    </section>
+  
+    <section class="featured-comedians">
+      <div class="s-title">
+        <h2>Featured Comedians</h2>
+        <hr>
+      </div>
+  
+      <ul class="comedians" id='comedians-list'>
+      </ul>
+
+      <div class="more-btn active" id="more-btn">
+        <p>MORE</p><i class="fa-solid fa-chevron-down"></i>
+      </div>
+    </section>
+  </div>
+
+  <div class="about-page" id="about-page">
+
+    <section class="about">
```

Comment:
- [ ] The requirement is that you have to create two seperate html files, the index.html and the about.html respectively, kindly do so.